tests/system/client.go

```diff
@@ -19,6 +19,11 @@ const (
 	waitInterval = 100 * time.Millisecond
 )
 
+var (
+	sessionStateFinal = regexp.MustCompile(`state:SESSION_STATE_TYPE_(COMPLETED|ERROR|STOPPED)`)
+	sessionStateAll   = regexp.MustCompile(`state:SESSION_STATE_TYPE_(COMPLETED|ERROR|STOPPED|CREATED|RUNNING)`)
```

Comment:
If we accept all states, we don't need to specify them.
```suggestion
	sessionStateAll   = regexp.MustCompile(`state:SESSION_STATE_TYPE_(\w+)`)
```

Reply:
Even better: `state:SESSION_STATE_TYPE_` without the suffix, since we don't check/use it in the code

Reply:
done